speech_recognition/__init__.py

```diff
@@ -914,8 +914,8 @@ def recognize_google(self, audio_data, key=None, language="en-US", pfilter=0, sh
         # return results
         if show_all:
             return actual_result
-        print('result2:')
-        pprint(actual_result, indent=4)
+        #print('result2:')
```

Comment:
MUST: Deleting print functions is more suitable rather than commenting them out.

See https://github.com/Uberi/speech_recognition/pull/651#pullrequestreview-1245753993

I think issue #652 will be resolved in pull request #651, so you don't need to include in this pull request.

Reply:
Totally fine.

I am just new to the whole process. I've been coding a long time and now just trying to give back to the community a little.

Closed the issue and opening a new pull request with the latest branch. Thanks for your patience.